src/fontMetrics.js

```diff
@@ -72,48 +70,55 @@ var ptPerEm = 10.0;
 // `\showthe\doublerulesep` in LaTeX.
 var doubleRuleSep = 2.0 / ptPerEm;
 
+// Return a function that will return the style specific value for the given
+// array of metric values.
+function createMetricsGetter(metrics) {
+    return function(style) {
+        var size = style.size;
+        if (size === Style.TEXT.size || size === Style.DISPLAY.size) {
+            return metrics[0];
+        } else if (size === Style.SCRIPT.size) {
+            return metrics[1];
+        } else if (size === Style.SCRIPTSCRIPT.size) {
+            return metrics[2];
+        }
+        throw new Error("Unexpected style size: " + style.size);
+    };
+}
+
 /**
  * This is just a mapping from common names to real metrics
  */
 var metrics = {
     xHeight: sigma5,
```

Comment:
All of the values for `sigma5` are the same.  Perhaps I should change this to an array and change this to `getXHeight` for consistency's sake.